src/player.cpp

```diff
@@ -1892,14 +1892,14 @@ void Player::removeExperience(uint64_t exp, bool sendText /* = false*/)
 	sendExperienceTracker(0, -static_cast<int64_t>(exp));
 }
 
-uint8_t Player::getPercentLevel(uint64_t count, uint64_t nextLevelCount)
+uint16_t Player::getPercentLevel(uint64_t count, uint64_t nextLevelCount)
 {
 	if (nextLevelCount == 0) {
 		return 0;
 	}
 
-	uint8_t result = (count * 100) / nextLevelCount;
-	if (result > 100) {
+	uint16_t result = ((count * 100.) / nextLevelCount * 100.);
```

Comment:
```suggestion
	uint16_t result = ((count * 10000.) / nextLevelCount);
```

Reply:
Are you sure about this?? Doesn't look right at first glance, of course I could be wrong

Reply:
Yes, the parenthesis here are not necessary since you only use multiplications (division is multiplication) and they already execute left to right. So

```cpp
((count * 100.) / nextLevelCount * 100.)
```

is equivalent to

```cpp
count * 100. / nextLevelCount * 100.
```

So you can just group 100 * 100 into 10000